src/api/index.ts

```diff
@@ -3,3 +3,4 @@
 
 export * from "./aptos";
 export * from "./aptosConfig";
+export { TransactionBuilderTypes } from "./transactionBuilder";
```

Comment:
why here? can't we just
`export * from "./transactions/transaction_builder";` in `src/index.ts`?

Reply:
pushed incorrect branch- fixed !